udata_front/theme/gouvfr/__init__.py

```diff
@@ -75,7 +75,7 @@
 theme.menu(gouvfr_menu)
 
 opendata_links = [
```

Comment:
I think you also need to update `The Open Data` to `Open Data` in https://github.com/etalab/udata-front/blob/731ffa1694c3a58bd8a989e4b1ac6add3f0ff1b6/udata_front/theme/gouvfr/templates/footer.html#L9